frontend/javascripts/admin/job/job_list_view.tsx

```diff
@@ -293,11 +297,23 @@ function JobListView() {
               cancelJob(job.id).then(() => fetchData());
             }
           }}
-          icon={<CloseCircleOutlined key="cancel" className="icon-margin-right" />}
+          icon={<CloseCircleOutlined className="icon-margin-right" />}
         >
           Cancel
         </AsyncLink>
       );
+    } else if (job.state === "FAILURE" && isCurrentUserSuperUser) {
+      return (
+        <AsyncLink
+          href="#"
+          onClick={async () => {
+            resumeJob(job.id).then(() => fetchData());
+          }}
+          icon={<PlayCircleOutlined className="icon-margin-right" />}
+        >
+          Resume
```

Comment:
maybe restart/retry is a better word here :thinking:?


```suggestion
          Retry
```

Reply:
I see your reasoning, I am just following the specification. @valentin-pinkau @daniel-wer  Opinion on this?

Reply:
It's a bit unclear, because it's actually both. Resume, because the tasks that already finished are skipped and retry, because the failed task is retried. Still, `Retry` is probably more accurate. Since it's only visible for super user that's fine I'd say. Otherwise one could add a tooltip, noting `Restarts the workflow from the failed task, skipping and reusing artifacts from preceding tasks that were already successful.`

Reply:
> Restarts the workflow from the failed task, skipping and reusing artifacts from preceding tasks that were already successful.

That sounds like a valuable addon regardless of this being only available to super users. As we might make this available to non super users in the future, we do not need to remember to add this tooltip. => I'd vote to add it here as this shouldn't be difficult and also should need a re-review thus not delaying merging this :)